mysql-test/suite/galera/t/galera_create_table_as_select.test

```diff
@@ -121,7 +125,7 @@ CREATE TABLE t1 AS SELECT * FROM t2;
 UNLOCK TABLES;
 
 --connection node_1
---error ER_TABLE_EXISTS_ERROR,ER_LOCK_DEADLOCK
+--error ER_TABLE_EXISTS_ERROR,ER_QUERY_INTERRUPTED
```

Comment:
The ER_LOCK_DEADLOCK -> ER_QUERY_INTERRUPTED change comes from changed BF abort path for operations which don't have open wsrep transaction. This should be fixed for CTAS specifically by opening a wsrep transaction for CTAS operation before MDL wait.